handlers/user.go

```diff
@@ -1627,15 +1627,18 @@ func UserDashboardWatchlistAdd(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	indices := make([]string, 0)
-	err = json.Unmarshal(body, &indices)
+	validators := make([]string, 0)
+	err = json.Unmarshal(body, &validators)
 	if err != nil {
 		logger.Errorf("error parsing request body: %v, %v", r.URL.String(), err)
 		ErrorOrJSONResponse(w, r, "Internal server error", http.StatusInternalServerError)
 		return
 	}
 	indicesParsed := make([]int64, 0)
-	for _, i := range indices {
+	for _, i := range validators {
+		if searchPubkeyExactRE.MatchString(i) {
+			continue
+		}
```

Comment:
Done, I assume an `OKResponse` is right here?

Reply:
Returning `200` might be wrong. Technically the endpoint expects the request to contain indices to perform some action on, while pubkeys get ignored. If the request does not contain any indices, it could get counted as a bad request, like if no request body had been sent at all. If we return `200`, users could be mislead to think that they successfully subscribed to something when in fact they didn't.

As a general rule, changing an API endpoint should not be problematic for users if the input parameters become LESS restrictive.

Reply:
Alternatively, if we want to avoid API behavior changes, we should revert to old behaviour i.e. not allow pubkeys in this function and make sure that the dashboards only sends present indices in the request or doesn't send the request at all if there are none.

Reply:
Sure, I reverted the changes in `UserDashboardWatchlistAdd` and only send it the indices.